flaml/autogen/agentchat/responsive_agent.py

```diff
@@ -87,6 +94,14 @@ def __init__(
                     If the code is executed in the current environment,
                     the code must be trusted.
                 - timeout (Optional, int): The maximum execution time in seconds.
+                - output_limit (Optional, int): default to -1 (no limit), if set, the "output_limit" will be triggered.
+                - on_output_limit(Optional, callable): The function to call when the 'output_limit' is exceeded.
+                    Note: if several code snippets are given in one reply, they will be executed one by one. The logs will be accumulated.
+                    Required for the "on_output_limit" function:
+                        1st Arg: (str, required) outputs from previous code snippets (will input an empty string if there is only one code snippet from the reply).
+                        2nd Arg: (str, required), output from current code snippet.
+                        Return: (str, required), the new output/message, it will replace the whole logs. You might want include the previous outputs in the new output if they are still needed.
+                    If None/ not passed, it will replace the output from current code block with an error message, and append it to previous logs.
```

Comment:
```suggestion
                    If None / not passed, it will replace the output from current code block with an error message, and append it to previous logs.
```